Source/WebCore/platform/ThreadGlobalData.h

```diff
@@ -106,6 +106,8 @@ class ThreadGlobalData : public PAL::ThreadGlobalData {
     FontCache* fontCacheIfNotDestroyed() { return m_destroyed ? nullptr : &fontCache(); }
 
 private:
+    bool m_destroyed { false };
+
```

Comment:
Nit: put this just before `m_cachedResourceRequestInitiators`?

Reply:
It's intentionally at the top, so that destruction order (last to first) guarantees that it's destroyed after the cache objects.

Reply:
I just mean in the same position relative to the other fields, but below the function declarations.

Reply:
I said the wrong field above: I mean above `m_isInRemoveAllEventListeners`.